skillpacks/review.py

```diff
@@ -19,6 +19,8 @@ def __init__(
         reviewer_type: str = ReviewerType.HUMAN.value,
         reason: Optional[str] = None,
         parent_id: Optional[str] = None,
+        correction: Optional[str] = None,
+        correction_schema: Optional[Dict[str, Any]] = None,    
```

Comment:
this is where we should take `Optional[BaseModel]` and then when we save it to the class call `correction_schema.model_json_schema()`

Reply:
Yea, I should've caught this. I saw how you did it in Taskara and then just forgot to follow it here.